UX_PRINCIPLES.md

```diff
@@ -0,0 +1,78 @@
+# Habitat CLI UX Principles
+
+## Commands
+### Be Judicious
+- Be judicious with the number of available commands
+- Limit the number of nouns - keeps the cognitive load small; users can memorize the core commands (keep them writing plans, not fumbling and looking up commands)
+### Build Consistency
+- Using the noun before the verb fosters consistency; you also get help at the noun level (imagine your command set as a tree hierarchy branching out from the root); if you can remember the nouns then you can quickly get the list of verbs for that noun (e.g. hab package -help)
+- Use shortcuts sparingly - they’re certainly useful but obstruct understanding and meaning
+- Don’t treat flags as 2nd class citizens. Be aware of existing flags and consider how they can/will be used across multiple commands and subcommands.
+### Make it Readable
+- Familiarity, simplicity and readability make it memorable. Docs are useful, but we don’t want people having to continually refer back to them.
+- Keep commands human readable and memorizable by using a sentence like structure `hab <noun> <verb>`
```

Comment:
I'm not sure how it would fit with your headings, but it feels like this would be better placed near the top, to give people context on Habitat's use of nouns and verbs before they are described in other principles.